swagger.yaml

```diff
@@ -294,7 +529,7 @@ paths:
           description: Forbidden
         404:
           description: Not found
-  /v1/ethereum/stake:
+  /api/v1/stake:
     post:
       summary: Stake x amounts of 32ETH on Ethereum
```

Comment:
Thanks for updating paths. Perhaps, summary shouldn't also mention Ethereum and ETH as it supposed to be chain-agnostic

Reply:
I'm not sure if the move from path to parameter was beneficial at all - it's much easier to document the different requirements in the different paths

Reply:
unless things are actually unified and there's no distinction